pkg/crc/api/api_client_test.go

```diff
@@ -213,6 +214,8 @@ func TestConfigGetAll(t *testing.T) {
 		switch v := v.Value.(type) {
 		case int:
 			configs[k] = float64(v)
+		case preset.Preset:
+			configs[k] = float64(v)
```

Comment:
cast to `float64`? This indicates a bug introduced by 80c53b7, the json serialization of the configuration should be `preset: "podman"`
I'd revert this part of 80c53b7:
```
diff --git a/pkg/crc/preset/preset.go b/pkg/crc/preset/preset.go
index 57cc3883..b038dda3 100644
--- a/pkg/crc/preset/preset.go
+++ b/pkg/crc/preset/preset.go
@@ -6,11 +6,11 @@ import (
        "github.com/code-ready/crc/pkg/crc/logging"
 )
 
-type Preset int
+type Preset string
 
 const (
-       Podman Preset = iota
-       OpenShift
+       Podman    Preset = "podman"
+       OpenShift Preset = "openshift"
 )
 
 func (preset Preset) String() string {
```